.github/workflows/render-auto-deploy.yml

```
@@ -0,0 +1,26 @@
+  push:
+    branches:
+      - master
+      - scout/sco-1-d886e992
+  workflow_dispatch:
+
+jobs:
+  deploy-render:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger backend deploy (Render)
+        if: ${{ secrets.RENDER_BACKEND_DEPLOY_HOOK != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)